tests/fxl-page-spread_left/OPS/package.opf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>When rendition:page-spread-left is applied to a spine item, it should be displayed on the left side of the synthetic spread.</dc:description>
+    <dc:identifier id="pub-id">fxl-page-spread_left</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>rendition:page-spread- left</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#page-spread_left</meta>
+    <meta property="dcterms:modified">2021-11-24T00:00:00Z</meta>
+    <meta property="rendition:layout">pre-paginated</meta>
+    <meta property="belongs-to-collection">should</meta>
```

Comment:
I anticipate @dlazin's comment on putting this entry in alphanumeric order, ie, before the `dcterms:modified` entry :-)